constructor/nsis/_nsis.py

```diff
@@ -47,8 +47,8 @@ def gui_excepthook(exctype, value, tb):
         import ctypes
         import traceback
         MB_ICONERROR = 0x00000010
-        title = u'Installation Error'
-        msg = u''.join(traceback.format_exception(exctype, value, tb))
+        title = "Installation Error"
```

Comment:
We should ignore this file.

It's executed at runtime by the Python interpreter included in the `base` env that the installer provides. If we pyupgrade this, we would prevent people from using `constructor` to deploy Py2k installations.

Not ideal, and the installer logic should be uncoupled from the installer contents (working on it via conda-standalone), but for now no modifications in this file.

Reply:
excluded